06_modules_lab/01.py

```diff
@@ -2,3 +2,13 @@
 from sys.argv import and prints "Hello Python"
 count times
 """
+
+import sys
+
+if len(sys.argv) > 1:
+    count = sys.argv[1]
+    for i in range(int(count)):
+        print "Hello Python"
+else:
+    program_name = sys.argv[0]
+    print "Usage: %s <count>" % program_name
```

Comment:
A better way to write this condition is to use `sys.exit()`. That also has the advantage to signal to the OS that something went wrong by returning an error code